modules/installation-gcp-permissions.adoc

```diff
@@ -40,6 +309,60 @@ ifdef::template[]
 .Required roles for user-provisioned GCP infrastructure
 * Deployment Manager Editor
 * Service Account Key Admin
+
+The following are the additional permissions required for user-provisioned GCP infrastructure for creating and deleting the {product-title} cluster.
+
+.Required Images permissions
+[%collapsible]
+====
+* `compute.images.create`
+* `compute.images.delete`
+* `compute.images.get`
+* `compute.images.list`
+====
+
+.Required permissions to get Region related information
+[%collapsible]
+====
+* `compute.regions.get`
+====
+
+.Required Deployment Manager permissions
+[%collapsible]
+====
+* `deploymentmanager.deployments.create`
+* `deploymentmanager.deployments.delete`
+* `deploymentmanager.deployments.get`
+* `deploymentmanager.deployments.list`
+* `deploymentmanager.manifests.get`
+* `deploymentmanager.operations.get`
+* `deploymentmanager.resources.list`
+====
+
+.Required Service Account permissions
```

Comment:
IPI part has the section already, so please remove the section for user-provisioned GCP infrastructure.

Reply:
Addressed- removed